pkg/auth/server/login/login.go

```diff
@@ -133,19 +133,36 @@ func (l *Login) handleLogin(w http.ResponseWriter, req *http.Request) {
 	l.auth.AuthenticationSucceeded(context, then, w, req)
 }
 
-var DefaultLoginFormRenderer = loginTemplateRenderer{}
+// NewLoginFormRenderer creates a login form rendered that takes in an optional custom template to
+// allow branding of the login page. If customLoginTemplateFile is not specified, uses the default.
+func NewLoginFormRenderer(customLoginTemplateFile string) (*loginTemplateRenderer, error) {
+	r := &loginTemplateRenderer{}
+	if len(customLoginTemplateFile) > 0 {
+		customTemplate, err := template.ParseFiles(customLoginTemplateFile)
+		if err != nil {
+			return nil, err
+		}
+		r.loginTemplate = customTemplate
+	} else {
+		r.loginTemplate = defaultLoginTemplate
+	}
+
```

Comment:
validation function here, called here and from the validator... there is work in progress to allow validating a configuration with a standalone command... would be nice if login template errors were picked up